ChartboostMediationAdapterAppLovin.podspec

```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |spec|
   spec.module_name  = 'ChartboostMediationAdapterAppLovin'
   spec.source       = { :git => 'https://github.com/ChartBoost/chartboost-mediation-ios-adapter-applovin.git', :tag => spec.version }
   spec.source_files = 'Source/**/*.{swift}'
+  spec.resource_bundles = { 'ChartboostMediationAdapterAppLovin' => ['Source/PrivacyInfo.xcprivacy'] }
```

Comment:
The privacy file is not a source file, should we place it one level up in the root folder?

Reply:
updated now